mainapp/templates/mainapp/request_form.html

```diff
@@ -102,6 +102,8 @@ <h3 id="myModalLabel">Allow Location - നിങ്ങളുടെ സ്ഥല
 					.done(function(response) {
 						var inputElem = $('<input class="form-control" id="id_latlng_display" readonly="true" value="' + response.display_name + '"</input>');
 						$(gpsField).before(inputElem);
+						$(gpsField).before('<br/><div id="map" style="height: 300px; width: 500px;"></div>');
```

Comment:
width of 500px is too big mobile screens.
I tried this change on it:

45 is magic number obtained by trial and error.
```
						var screenWidth = window.innerWidth - 45;
						if (screenWidth > 500) {
							screenWidth = 500;
						}
						$(gpsField).before('<br/><div id="map" style="height: 300px; width: ' + screenWidth + 'px;"></div>');
```